pkg/frontend/frontend_test.go

```diff
@@ -0,0 +1,39 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/gorilla/mux"
+)
+
+func TestURLPathsAreLowerCase(t *testing.T) {
+	f := &frontend{}
+
+	fakeRouter := mux.NewRouter()
+	f.unauthenticatedRoutes(fakeRouter.NewRoute().Subrouter())
+	f.authenticatedRoutes(fakeRouter.NewRoute().Subrouter())
+
+	varCleanupRe := regexp.MustCompile(`{.*?}`)
+	err := fakeRouter.Walk(func(route *mux.Route, _ *mux.Router, _ []*mux.Route) error {
+		pathTemplate, err := route.GetPathTemplate()
+
+		// Ignore the error: it can occur when a route has no path,
+		// but there is no way to check it here
+		if err == nil {
```

Comment:
nit: `if err != nil { continue }` reduces indentation

Reply:
It's not a loop, but I changed it in a similar way with `return nil`